src/LibraryManager/Microsoft.Web.LibraryManager.csproj

```diff
@@ -11,10 +11,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="System.ComponentModel.Composition" Version="4.5.0" />
```

Comment:
@alexgav I couldn't find any good use case for the Microsoft.VisualStudio.Composition Nuget package so I just stuck with using this.